lib/dependencies.js

```diff
@@ -44,6 +44,8 @@ let versionMap = {
   "karma-babel-preprocessor": "^6.0.1",
   "karma-typescript-preprocessor": "^0.2.1",
   "minimatch": "^3.0.2",
+  "requirejs": "^2.3.2",
+  "text": "github:requirejs/text",
```

Comment:
> If you don't specify a commit-ish then master will be used.
> ―npm install docs

There is the [`latest`](https://github.com/requirejs/text/releases/tag/latest) tag which should be more stable then `master`.
That would be `"text": "github:requirejs/text#latest",`.

Sorry for not catching this in my first review.